### ISSUE TITLE
File has changes immediately after opening the file

### ISSUE THREAD

**Issue by fultonm** [excalidraw/excalidraw-vscode#129]
After opening an excalidraw file in VsCode, the file always has changes even if the file has not really been touched yet.

Short screen recording demonstrating the behavior: https://1drv.ms/v/s!AiSCz54h8oiP2DWiRl2ThvtWDyzw?e=G9KR5u 

I believe this issue starts to occur when objects are grouped. Without grouping, the file doesn't have changes immediately after opening.

This makes it difficult to know which files have *actually* been modified, and also creates some anxiety about clicking save/discard changes. Since I am not sure if the file has been ghost-modified from this bug or if the file really had changes.

To figure out what changes there were, I initialized a git repository, and found the version, versionNonce, and updated fields were all modified.

So a potential fix could have the extension wait until the file has been modified by the user before changing these fields.

**Comment by fultonm** [comment]
Is this a known issue?

Can others reproduce?

**Comment by EnzoPV25** [comment]
Hello,

I have the same thing happening on my end. 

I just created an folder with some .excalidraw files, and when opening them in VScode they appear with changes.

Is there any workaround ?

**Comment by fultonm** [comment]
I think I noticed Excalidraw for web also does this. So this may be an upstream issue.

can anyone confirm?

**Comment by asinyagin** [comment]
I have the same behavior. Excalidraw changes fields `version`, `versionNonce`, `updated` of each element. It's hard to read diff of files because of these changes.

**Comment by lyrixderaven** [comment]
I have exactly the same issue; as soon as I've opened a file it immediately presents with changes; diffing the raw text source shows precisely the fields mentioned by @asinyagin have changed. Any idea on how to resolve this? 

We'd like to use Excalidraw extensively for a variety of visualization tasks, and share progress via git, but this problem leads to lots of merge issues due to committing files and changes that have not actually been changed.

**Comment by mselmanyildirim** [comment]
Seems duplicate of #81 and #94 . It's been a while without fix. I wish it would be fixed, so that version tracking becomes easy.

**Comment by pomdtr** [comment]
Yeah sorry about that. I plan to resume development at some point, but I don't have much free time these days. I would love some support on this (that's was my motivation for moving the repo to the excalidraw org).

**Comment by fultonm** [comment]
I fully planned to fix this when I opened the ticket, but ya, time and life and things 😭 also it looked like it was more complicated of a problem than it looks

**Comment by makersky** [comment]
I have the same issue, in my case it's always "type": "text" objects and "version", "versionNonce", "updated" fields that changes.

BTW. Excalidraw is awesome. Excalidraw VSCode is awesome. It was such a relief when I could finally stop using Visio and switch to something that is so much simpler, so much better and can be used with git. Great work!
